Location: RuntimeExceptionAutoPropagate.java

Content: 

public void method1(){

  try {

    method2();

  }

 catch (  ArithmeticException e) {

    System.out.println("****ArithmeticException handled. ****");

    e.printStackTrace();

  }

}

Location: RuntimeExceptionAutoPropagate.java

Content: 

public void method2(){

  method3();

}

Location: RuntimeExceptionAutoPropagate.java

Content: 

public void method3(){

  int a=75 / 0;

}

